addons/web/static/src/js/framework/misc.js

```diff
@@ -204,6 +204,39 @@ function ReloadContext (parent, action) {
 }
 core.action_registry.add("reload_context", ReloadContext);
 
+
+/**
+ * Generic widget to create an iframe inside the backend
+ * that listens for clicks to interact with the backend.
+ *
+ * It should be extended by overwritting the methods:
+ *      init: function(parent) {
+ *          this._super(parent, <url_of_iframe>
+ *      },
+ *      iframe_clicked: function(e){
+ *          filter the clicks you want to use and apply
+ *          an action on it
+ *      }
+ */
+var IframeBackend = Widget.extend({
+    tagName: 'iframe',
+    init: function(parent, url) {
+        this._super(parent);
+        this.url = url;
+    },
+    start: function() {
+        this.$el.attr({'style': 'height: 100%; width: 100%; border: 0;', 'src': this.url});
```

Comment:
a slightly better way to do this would be:
this.$el.css({height: '100%', width: '100%', border: 0});
this.$el.attr({src: this.url});